app/jobs/load_repositories_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class LoadRepositoriesJob < ApplicationJob
+  queue_as :default
+
+  def perform(organization)
+    load_repositories(organization)
+  end
+
+  private
+
+  def load_repositories(organization)
```

Comment:
Once again. please make this class-level method, it doesn't use anything on the instance-level (and rightly so)